### ISSUE TITLE
It should be possible to close panes using Esc

### ISSUE THREAD

**Issue by indeyets** [FreeFeed/freefeed-react-client#1162]
1. "Subscribe" pane on user's page
2. "Add friend list" pane on "Browse/edit …" page

are there more of these?

Whenever user presses "Esc" button, the pane should be closed. If pane is in it's initial unchanged state it should happen instantly. If pane's form was modified by user then confirmation dialog should be shown before closing

**Comment by indeyets** [comment]
Works as expected
